hedera-mirror-rest/__tests__/specs/contracts/results/state/no-params.json

```
@@ -0,0 +1,42 @@
+{
+  "description": "Contract state api call for a contract",
+  "setup": {
+    "contractStates": [
+      {
+        "created_timestamp": "1676540001234390005",
+        "modified_timestamp": "1676540001234390005",
+        "contract_id": 5001,
+        "slot": "01",
+        "value": "01"
+      },
+      {
+        "created_timestamp": "1676540001234390005",
+        "modified_timestamp": "1676540001234390005",
+        "contract_id": 5001,
+        "slot": "02",
+        "value": "02"
+      }
+    ]
+  },
+  "urls": ["/api/v1/contracts/5001/state"],
```

We should test multiple path params:
```suggestion
  "urls": [
    "/api/v1/contracts/0.0.5001/state",
    "/api/v1/contracts/0.5001/state",
    "/api/v1/contracts/5001/state",
    "/api/v1/contracts/62cf9068fed962cf9aaabbb962cf9068fed9dddd/state",
    "/api/v1/contracts/0.62cf9068fed962cf9aaabbb962cf9068fed9dddd/state",
    "/api/v1/contracts/0.0.62cf9068fed962cf9aaabbb962cf9068fed9dddd/state",
    "/api/v1/contracts/0x62cf9068fed962cf9aaabbb962cf9068fed9dddd/state"
  ],
```